documentation/itemdatalist.txt

```diff
@@ -3,6 +3,8 @@ nameDe, overlay generic item name
 nameEn, overlay generic item name
 descriptionDe, overlay generic look at
 descriptionEn, overlay generic look at
+qualityAtCreation, best possible quality, set if item was repaired
```

Comment:
Is this data field also being set when the item is actually crafted? Based on the name, it seems like it should be.